.github/workflows/velox_be.yml

```diff
@@ -59,7 +59,8 @@ jobs:
           docker cp ~/.m2/repository velox-backend-ubuntu2004-test-$GITHUB_RUN_ID:/root/.m2/
       - name: Build Gluten Velox-backend third party
         run: |
-          docker exec velox-backend-ubuntu2004-test-$GITHUB_RUN_ID bash -c '
+          docker exec velox-backend-ubuntu2004-test-$GITHUB_RUN_ID bash -ec '
+          source /opt/gluten/dev/vcpkg/env.sh && \
```

Comment:
Should we remove `source /opt/gluten/dev/vcpkg/env.sh` here. because we do not use vcpkg in CI.